BUILD.md

```diff
@@ -89,11 +89,17 @@ The `npm run watch` script in action:
   * Windows Only
     * WSL2 or any virtual machine running Ubuntu 20.04 LTS
   * Git
-  * NodeJS 16, npm 7 (we recommend using the Node Version Manager (nvm) if available for your OS)
+  * NodeJS v16.13.1, npm v8.1.2 (we recommend using the Node Version Manager (nvm) if available for your OS)
+    ```
+    nvm install 16.13.1
+    nvm use 16.13.1
+    ```
   * Yarn 
     * `npm run install-yarn` (from within the project directory)
   * Docker Engine
   * Docker Compose
+  * OpenJDK
+    * `sudo apt install openjdk-17-jre-headless`
```

Comment:
@ruzell22 This would install the Java Runtime Environment not the Java Development Kit which is not enough unless you are an end user running a Java based application. I think the one you want is probably `openjdk-11-jdk` or maybe `openjdk-11-jdk-headless` 

The reason I'm saying 11 instead of 17 is because I'm not sure if 17 can build the Corda/Springboot code components. If you could verify that it works with `openjdk-17-jdk` then I'd be fine bumping it up to 17 as well so just let me know.